src/components/util/getDataMixins.js

```diff
@@ -0,0 +1,131 @@
+import $ from 'jquery';
```

Comment:
This mixin implements a common data retrieval pattern that showed up across a handful of components. It can easily be used in conjunction with the `DataLoader.vue` component, as it defines variables that that component needs.